scipio/src/timer/time.rs

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the
```

Comment:
Much like the previous patch, this one has to go too.

We had this conversation in private, but for the benefits of others:

`Instant::now()` does not take a mutex for no reason. In platforms where timekeeping is stable it doesn't take the mutex at all. And where it is not (something like) the mutex is needed anyway.

Reply:
Wiped